admin/app/components/certification-centers/information-edit.js

```diff
@@ -1,16 +1,21 @@
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
```

Comment:
Je met mon retour sur cette ligne car c'est en lien avec l'édition d'un centre de certification mais je n'ai encore la cause du soucis.

Je modifie un centre de certification initialement V3. On peut voir dès le début que le tag du V3 est grisé bien que l'API retourne un true sur le v3.
Les modifications des trois champs DPO ne semblent pas être pris en compte également.


https://github.com/1024pix/pix/assets/58915422/4114df83-5280-415a-8493-00397e82de6a

<img width="1132" alt="Capture d’écran 2024-06-10 à 10 37 02" src="https://github.com/1024pix/pix/assets/58915422/3ef8d6ce-97e4-45ea-834f-185d0e9c3df6">

Reply:
Ah 🤔 
Je vais regarder ça ! Merci en tout cas, et top avec la vidéo ça permet de vite retrouver 🙏

Reply:
Ok déjà concernant le non retour des infos DPO, c'est HS à cette PR mais il manque un await sur la récupération des infos DPO lors du get 🥲
https://github.com/1024pix/pix/blob/3e4741ad961a693a981fa1a20625b0daf8d61eed/api/src/certification/enrolment/domain/usecases/get-center-for-admin.js#L5-L5

Reply:
**Piste** : json-api attend un type du tableau des habilitations qu'il ne reçoit ? 

<img width="300" alt="Capture d’écran 2024-06-10 à 14 37 35" src="https://github.com/1024pix/pix/assets/58915422/48655459-5a52-4fc7-9df5-dca2281643ca">

Reply:
Histoire que ce soit noté ici : J'ai l'impression que le await dans le usecase corrige tout le fonctionnel 😄 !